lib/diagnostic.js

```diff
@@ -39,20 +52,39 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+
+let pet = {
+  name: 'doggy',
+  species: 'canine',
+  breed: 'labradoodle',
+  vocalize: function () {
```

Comment:
You want this function to return a property on the `pet` object, rather than returning a variable that you define inside the function itself. For example, you would want to include a property called vocalization: `vocalization: 'woouuaafuh'` and then have your `bark` method look like this: `vocalize: function(){return this.vocalization}`